plugins/utilities.json

```diff
@@ -3,6 +3,25 @@
   "description": "Utilities",
   "plugins_base_url": "http://github.com/bombsquad-community/plugin-manager/{content_type}/{tag}/plugins/utilities",
   "plugins": {
+    "mood_light": {
+      "description": "mood lighting in co-op games",
+      "external_url": "https://github.com/Loup-Garou911XD/plugin-manager",
```

Comment:
this field was planned to link to where users can find more info about this specific plugin, say a blog post or a yt video

Reply:
So shall i remove it?

Reply:
Or maybe create a .md file?

Reply:
yep, i think you should, current url isn't very specific to only this plugin
although `external_url` isn't being used anywhere in the plugin manager right now, but it's something that'll be nice to have in future

Reply:
I'll remove it then for now